scalajs/src/main/scala/org/scalablytyped/converter/internal/scalajs/flavours/GenCompanions.scala

```diff
@@ -52,7 +55,9 @@ final class GenCompanions(findProps: FindProps) extends TreeTransformation {
 
               case Res.One(_, props) if props.isEmpty => Empty
               case Res.One(_, props) =>
-                val requiredProps = props.filter(_.optionality === Optionality.No)
+                val requiredProps =
```

Comment:
Can you flip these two `if`s as well? `if (enableLongApplyMethod) props else ...`

Reply:
Fixed.